cmd/pj-rehearse/server.go

```diff
@@ -135,10 +141,14 @@ func serverFromOptions(o options) (*server, error) {
 	rehearsalConfig.ProwjobNamespace = c.ProwJobNamespace
 	rehearsalConfig.PodNamespace = c.PodNamespace
 
+	mirrorStore, quayIOImageHelper := mirrorstoreserver.MirrorStoreServer(6378, time.Minute*1, o.registryConfig, true, false)
+
 	return &server{
-		ghc:             ghc,
-		gc:              gc,
-		rehearsalConfig: rehearsalConfig,
+		ghc:               ghc,
+		gc:                gc,
+		rehearsalConfig:   rehearsalConfig,
+		mirrorStore:       mirrorStore,
+		quayIOImageHelper: quayIOImageHelper,
```

Comment:
Here I added the `MirrorStoreServer` to the already existing `pj-rehearse` server and propagated it to the `RehearseJobs` method